antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+set -e
+
+CURRENT_BRANCH=${git describe --tag}
+
+createReleaseDocumentation() {
+  chmod a+x createOrUpdateVersionDocumentationFolder.sh
+  sh ./createOrUpdateVersionDocumentationFolder.sh ${CURRENT_BRANCH}
+  git checkout ${CURRENT_BRANCH}
+}
+
+deleteOldSnapshotFolderInGhPages() {
+  git checkout gh-pages
+  rm -rf "*SNAPSHOT"
+  git commit -am "remove old snapshot documentation folder"
+  git checkout ${CURRENT_BRANCH}
+}
+
+createCurrentSnapshotDocumentation() {
+  chmod a+x createOrUpdateVersionDocumentationFolder.sh
+  sh ./createOrUpdateVersionDocumentationFolder.sh master
+  git checkout ${CURRENT_BRANCH}
+}
+
+createNewIndexRootFile() {
+  chmod a+x writeIndexRootFile.sh
+  sh ./writeIndexRootFile.sh
+}
+
+pushChanges() {
+  git push `git config remote.origin.url | sed "s/https:\/\//https:\/\/${GITHUB_USER}:${GITHUB_PASSWORD}@/g"` gh-pages
```

Comment:
Sometimes you use `$()` and sometimes backticks for command expansion. Aligning on either one would be good, preferably `$()` as backticks can easily be confused with single quotes.

Reply:
But this is probably not needed anyway as you can explicitly push to a fully qualified URL:

    git push https://${GITHUB_USER}:${GITHUB_PASSWORD}@... gh-pages